libs/tokens/src/utils/mergeTokenMaps.ts

```diff
@@ -5,15 +5,18 @@ import { TokensMap } from '../types'
  * Earlier® maps in the list take precedence for basic properties, but tags are concatenated
  */
 export function mergeTokenMaps(...tokenMaps: (TokensMap | null | undefined)[]): TokensMap {
+  // Note: Earlier maps in the original list (processed later here)
+  // have their properties OVERRIDDEN by later maps in the original list (processed earlier here).
```

Comment:
This comment conflicts with the docstring. I did that so my bad, forgot to update it.
Can you update the docstring and remove the comment instead?

Reply:
Addressed now.